src/native-common/Image.tsx

```diff
@@ -194,37 +195,46 @@ export class Image extends React.Component<Types.ImageProps, ImageState> impleme
         }
     }
 
+    private _buildHeaders(): Types.Headers|undefined {
+        if (this.props.headers) {
+            const cacheControlHeader = this._getMaxStaleHeader();
+            if (cacheControlHeader) {
+                // Filter out Cache-Control: max-stale. It has the opposite effect on iOS: instead of having
+                // the cache return stale data it disables the cache altogether. We emulate the header by
+                // retrying with cache: 'only-if-cached'.
+                const headers = _.clone(this.props.headers);
```

Comment:
seems that `omit` is also acceptable in this case. 
```typescript 
return _.omit(this.props.headers, [cacheControlHeader])
```
what do you think?

Reply:
Nice and concise, thanks!